telemetry/definitions/commonDefinitions.json

```diff
@@ -3430,23 +3473,50 @@
             "passive": true
         },
         {
-            "name": "codeTransform_buildComplete",
-            "passive": true,
-            "description": "During the transformation progress, log the status of the build.",
+            "name": "codeTransform_dependenciesCopied",
+            "description": "Deprecated - use codeTransform_uploadProject (The repo copies over at least one dependency successfully)",
             "metadata": [
                 {
-                    "type": "buildDuration",
+                    "type": "codeTransformSessionId",
                     "required": true
                 }
             ]
         },
         {
-            "name": "codeTransform_dependenciesCopied",
-            "description": "The repo copies over at least one dependency successfully.",
+            "name": "codeTransform_downloadArtifact",
+            "description": "Transform or user downloads code transform artifact",
```

Comment:
When you transform or user downloads, do mean can be active or passive according to use case?

Reply:
I think it will be better to change to user download, so I'll update it.